### ISSUE TITLE
Revisit standard settings discussion: I don't know what layer height I _should_ use?

### ISSUE THREAD

**Issue by Sloloem** [kickstarter/kickstarter-autodesk-3d#28]
Relating to issues: #7 #8 #10 #27 

It's...fairly difficult to gain a baseline level of understanding of the 3D printing world, and adding information like what is being asked for in these issues to a benchmark project like this would go a long way to helping create that easily accessible reference of basic knowledge that often permeates other hobbies.

If you'll pardon me to plead my case:

Having just setup my printer, done a few of the pre-compiled models that had prebuilt gcode bundled on an SD card, I decided I wanted to try this kickstarter test shape and compare to the known score for the model printer I have so I can see how well I've set the thing up as compared to the factory-assembled version.  Unfortunately no one seems willing or able to provide the standard settings that were used to slice the model when creating that benchmark.

In a comment for closing #10, @andreasbastian wrote "Standard settings are the settings that a printer/slicer/materials manufacturer/provider prescribe for using their product." but this doesn't actually answer any of the questions being asked or help understand how to use the model.  I've consulted every manual I can find for my printer and slicer, and nowhere does anything suggest an "optimal", "default", or "standard" layer height, infill %, infill pattern, or value for any parameter that might impact the quality or time taken to print.

I use PrusaSlicer and this is further complicated by several fairly equally recommended quality profiles ranging from "Ultradetail" to "Draft".  These profiles vary estimated print times between 2 hours and 22 hours.  The layer height settings of these profiles varies between 0.05mm and 0.3mm.  Whether supports are printed is also up in the air but doesn't vary by profile.

I _think_ they should be printed without support material and that adding supports would defeat the point of the overhang and bridging tests, but no one will write anything down to confirm or deny, officially, how I'm supposed to print this model.  The only real evidence I have is the DrVax youtube videos comparing Ender 5 and Prusa MK3 where he shows removing the print from the sheet.  In his video he doesn't have any support material on these, which are implied to be fresh from the printer.

But presumably layer height resolution can also impact how a lot of model features score like fine flow control, negative space, maybe even bridging and overhangs but I'm just guessing.  So when the readme says something like "Prusa i3 MK3 scores 22.5"...what settings were used to get that score?  At very least what basic profile or layer height?  If you included how long it took to print I could at least pick the quality profile that's nearest to that length but as it stands I am completely in the dark here.

Obviously there is almost infinite tweakability in printer and slicer settings but just by picking a different built-in profile and not knowing anything I can make a Prusa take over a day to print this model, but no one will say whether that's _the right way to do it_.  And measuring print time like suggested in #8 is almost useless without this baseline assumption of the resolution the model is being printed at since even small changes to that layer height value change the print time dramatically.

tl;dr:  Please for Pete's sake, just include the most basic print parameters for most slicers in the readme, it'll help out the newbies immeasurably and really increase consistency in benchmark data.

**Comment by Rayth** [comment]
> nowhere does anything suggest an "optimal", "default", or "standard" layer height, infill %, infill pattern, or value for any parameter that might impact the quality or time taken to print.

This is because you can't be given settings like this. As with resin printing, FDM printers are all different and should really be calibrated. 2 People could have the same printer, same filament, same slicer, same model and STILL have different settings. Printers need to be calibrated.
